builder/ebs/fast_launch_setup.go

```diff
@@ -0,0 +1,106 @@
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type FastLaunchConfig
+
+package ebs
+
+import "fmt"
+
+// FastLaunchConfig is the configuration for setting up fast-launch for Windows AMIs
+//
+// NOTE: requires the Windows image to be sysprep'd to enable fast-launch. See the
+// AWS docs for more information:
+// https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/win-ami-config-fast-launch.html
+type FastLaunchConfig struct {
+	// Configure AMI with fast-launch on Windows AMIs
+	UseFastLaunch bool `mapstructure:"enable_fast_launch"`
+	// The ID of the launch template to use for the fast launch
+	//
+	// This cannot be specified in conjunction with the template name.
+	//
+	// If no template is specified, the default launch template will be used,
+	// as specified in the AWS docs.
+	LaunchTemplateID string `mapstructure:"template_id"`
+	// The name of the launch template to use for fast launch
+	//
+	// This cannot be specified in conjunction with the template ID.
+	//
+	// If no template is specified, the default launch template will be used,
+	// as specified in the AWS docs.
+	LaunchTemplateName string `mapstructure:"template_name"`
+	// The version of the launch template to use
+	//
+	// If unspecified, and a template is referenced, this will default to
+	// the latest version available for the template.
+	LaunchTemplateVersion int `mapstructure:"template_version"`
+	// Maximum number of instances to launch for creating pre-provisioned snapshots
+	//
+	// If specified, must be a minimum of `6`
```

Comment:
I see this answer my question from above. Is there a default value if not specified?